docs/user-guide/map-views.md

```diff
@@ -0,0 +1,77 @@
+
+<img src="../img/map-views/position.jpg" class="ms-docimage"/>
+
+* Modify the duration of the animation and enable/disable the transition during the [Views Navigation](map-views.md#3d-views-navigations) by clicking on **Animation** section.
+
+<img src="../img/map-views/animation.jpg" class="ms-docimage"/>
+
+* Select a WFS or Vector layer available in the map to create a mask on all the 3D tiles visible in the map by clicking **Mask** section. The WFS or Vector layer is added to the map by the user and then used the layers features in a transparent way to create the clipping area.
+
+<img src="../img/map-views/mask-panel.jpg" class="ms-docimage"/>
+
+If the user enables the **Inverse** option, a hole is created inside the 3D Tile layers. This hole can be filled by only 3D objects (3D Tiles cannot be used to fill these holes because all 3D tiles will be masked).
+
+<img src="../img/map-views/mask-inverse.jpg" class="ms-docimage"/>
+
+If the user disables the **Inverse** option the clipping style is used for the view, then the selected WFS layer masks the 3D Tile layers outside its polygon.
+
+<img src="../img/map-views/mask.jpg" class="ms-docimage"/>
+
+* Enable the translucency of the globe in **Globe Translucency** section so that it is possible to see layers beneath the globe's surface.
+
+<img src="../img/map-views/translucency.jpg" class="ms-docimage"/>
+
+* Customize the layer present on the view by clicking **Layer Options** section. The user can simply enable or disable all the layers present on [TOC](toc.md#table-of-contents) and change the opacity of the layer.
+
+<img src="../img/map-views/layer-options-tool.jpg" class="ms-docimage"/>
+
+!!! warning
+    In the *2D mode*, the *3D Tiles* layer and the *Terrain* are not displayed in the *Layer Options*.
+
+For the *3D mode*, it is also possible to clip each layer of 3D tiles using another layer. For this operation, the user can select a WFS or Vector layer available in the map from the *Clipping layer source* dropdown menu and choose a *Clipping feature* of the layer from the dropdown list where the hole on the 3D Tiles will be created.
+
+<img src="../img/map-views/clipping.gif" class="ms-docimage"/>
+
+!!! warning
+    The WFS layer can be only polygon feature and concave polygon are not supported by this type of clipping, only convex layers can be used from this purpose.
+
+!!! note
+    Unlike the *Mask* option, described above, the *Clipping* area is a more narrowly focused tool because:
+
+    * The clipping are is visible only on the selected 3D Tiles layer and not on the whole view
+
+    * It is possible, using the *Clip function*, to select the feature of the WFS or Vector layer's attribute table that you want to use to do the clipping
```

Comment:
* It is possible to select the feature of the WFS or Vector layer that you want to use for the clipping